### ISSUE TITLE
High Cloud Download usage

### ISSUE THREAD

**Issue by sasagr** [dgalli1/DockerPremiumize#3]
Since a few days I m noticing an high premiumize Cloud Download usage and similarly also a local high bandwidth consumption. I m trying to localize the source of this usage because I m not streaming as much as I can see in the above usage. I can only say that I added a few more movies and shows recently in Radarr/Sonarr.
I m wondering if the rclone mounting settings could have an heavy impact on this, maybe the cache settings?
Otherwise pls advise if you have any idea.

Thanks

**Comment by sasagr** [comment]
I m monitoring the internet bandwidth by processes (nethogs) and seeing random peaks with rclone, 2-4 Mb/sec without any streaming and in the same time Cloud Download usage is increasing. In radarr/sonarr logs I dont see any task running.
What is rclone doing?

**Comment by dgalli1** [comment]
How often? Probably its just rclone.
Otherwhise just make sure that https://github.com/dgalli1/DockerPremiumize#my-whole-traffic-is-gone
This option is checked correctly in sonarr & Radarr

**Comment by sasagr** [comment]
Quite often... I will try to mount only with these arguments: --allow-other --allow-non-empty --log-level INFO
Read only is unticked in my case (this is only present for torrent and not usenet client) !!! Wrong or right?

**Comment by sasagr** [comment]
ok, I made sure Read Only in CHECKED in both radarr and sonarr and mounted rclone with your suggested arguments. Will keep an eye on the bandwidth...

**Comment by sasagr** [comment]
I do still have the issue.. I m noticing an high Cloud Download usage. Just today without any streaming I have already 37Gb. 
Looking at the history before I met this repo I had 2-5Gb then I see from beginning of November when I started using sonarr it increases to 10-20Gb.. since end of last week I have min 30Gb with pick of 155Gb.. The only difference in this last period is that I started using also Radarr, that I have a added a few movies and serie (not so many anyway) and I did update Radarr/Sonarr. I m going crazy trying to understand what is causing such increase... With this setup it cannot work. I need to stay max 30Gb per day if I want to keep the PM account. In the same time I m monitoring also the device (raspberry pi) where the repo is installed and I see pick of downloads... mainly from rclone process. I see that every night at 4:00am there is a pick of download (not sure if it is still rclone but probably it is). Any idea what else could cause such over usage?
If not I m going to start to shutdown services and see which one/s is causing it

**Comment by sasagr** [comment]
by the way... I also noticed that today I have a torrent usage of 5.92Gb.. as far as I can see today it just downloaded a file of 3Gb but I also noticed that every time something is in queue in the Activity tab, it starts to show from history than it goes to queue and it shows in two lines? Is it downloading it 2 times?? maybe this is because I see 5.92 instead of 3 ??

**Comment by sasagr** [comment]
![Screenshot 2020-11-18 202315](https://user-images.githubusercontent.com/3427877/99571399-ed417a00-29db-11eb-931e-b6a10de0d052.jpg)
is it normal?

**Comment by sasagr** [comment]
I think now I understood how it works. Everytime I add a movie and the movie is grabbed and cached in premiumize then it is downloaded by rclone locally fully in my device in a tmp folder (this has 1:1 impact on Cloud Download usage and my internet bandwidth and also on my device storage). Once the movie is downloaded or actually even before the download is completed, I think radarr processes it and it starts to upload it again to the radarr folder removing it from the sonar_download folder. This last step has no impact on the Cloud Download (probably) but has impact on my internet bandwidth usage. So still did not watch the movie but it has been already counted in PM usage once and in my internet twice.
Actually I noticed that the device keep on downloading "something" even even over the size of the movie and it stops only when the upload has been completed. This has to do with the READ ONLY setting? Read-Only is CHECKED (enabled) for me.
I also noticed that the Queue at the end it shows still one line with "Failed to import, destination already exist" and actually the file has been imported.
Are all these behavior normal? Pls advise

**Comment by dgalli1** [comment]
Is the Setting analyze Video unter /settings/mediamanagement checked?

**Comment by sasagr** [comment]
Yes. I had it in radarr. Immediately disabled it. 
Otherwise the setting READ ONLY checked abd USE HARDLINK INSTEAD OF COPY checked are correct?
And to answer my previous question, is in any case rclone supposed to download locally the file?

**Comment by sasagr** [comment]
after chaging the Analyze video setting I did another test with a movie of 800mb and it happened the same thing.. rclone keep on downloading locally the file (even above 800mb) waiting to be fully uploaded. I deleted the movie in radarr from queue and movie list and rclone keeps on working. I will check if it happens the same with sonarr. I checked my logs at night and rclone between 4 and 6 am downloaded about 10Gb. I have no idea what it did. I have no new releases downloaded.

**Comment by sasagr** [comment]
With sonarr it is happening exactly the same.
At this point I did some test changing the settings of READ ONLY and USE HARDLINK INSTEAD OF COPY.
READ ONLY unchecked. The file is not downloaded locally and the cloud/internet usage is very limited. So this seems the right setting for me. Maybe you meant it like this?
USE HARDLINK INSTEAD OF COPY checked or unchecked does not make big difference. Any suggestion here?

**Comment by sasagr** [comment]
I found to be BAZARR that is consuming a lot of bandwidth between 4 and 6am... as matter of fact there are by default tasks to scan to update all episodes and movies for subtitles from disk. I see a lot of activity with rclone in that period and it consumes both PM cloud usage and internet bandwidth (at least 20-30Gb) with eventually zero results. I have no idea what it does and why it affects usage. I deleted these tasks and I have no over usage anymore. Also as mentioned before READ ONLY is also unchecked.
